util/liblog/liblog.go

```diff
@@ -11,13 +11,18 @@ type Logr interface {
 }
 
 func FromSDKLogger(l log.Logger) Logr {
-	return &lgwr{l: l}
+	return &lgwr{l}
 }
 
 type lgwr struct {
 	l log.Logger
 }
 
+// Impl implements Logr.
+func (*lgwr) Impl() any {
+	return log.NewNopLogger()
+}
+
```

Comment:
Unneeded.

Reply:
but if we don't implement it we are getting an error that Impl() function is not implemented.